src/js/rebuses.js

```diff
@@ -254,6 +254,18 @@ const rebuses = [
   {
     symbols: ['🐱', '+', 'er', '+', '🗼'],
     words: ['caterpillar']
+  },
+  {
+    symbols: ['🧢', '+', 'tain'],
+    words: ['captain']
+  },
+  {
+    symbols: ['🦊', '+', 'hole'],
+    words: ['foxhole']
+  },
+  {
+    symbols: ['🐦', '+', 's-' + '👁️'],
```

Comment:
The words in the last two rebuses are the same, is that your intention? This also produces a lint error (no-useless-concat) caused by the string concatenation in `symbols: ['🐦', '+', 's-' + '👁️']`